src/pages/404.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import Link from "next/link";
+
+import { Box, Button, Container, HStack, Text } from "@chakra-ui/react";
+
+function NotFound() {

```

Comment:
Generally speaking its good practice to name `page` components in such a way that it shows it's a "_page component_". So, can you rename it something like this - `NotFoundPage`?